saleor/dashboard/product/__init__.py

```diff
@@ -0,0 +1,11 @@
+from django.utils.translation import pgettext_lazy
+
+
+class ProductBulkAction:
+    """Represents types of product bulk actions handled in dashboard."""
+    PUBLISH = 'Publish'
+    UNPUBLISH = 'Unpublish'
+
+    CHOICES = [
```

Comment:
Is there a good reason why CHOICES are not dictionary?
Also, when we finally drop Python2 support we might consider using enums.

Reply:
We tend to use this pattern with other constants and choices as well and I think we should stick to it. It allows you to pass `ProductBulkAction.CHOICES` to ChoiceFields.